kernel/aster-nix/src/time/clocks/mod.rs

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use int_to_c_enum::TryFromInt;
+pub use system_wide::*;
+
+mod system_wide;
+
+#[derive(Debug, Copy, Clone, TryFromInt, PartialEq)]
+#[repr(i32)]
+pub enum ClockID {
+    CLOCK_REALTIME = 0,
+    CLOCK_MONOTONIC = 1,
```

Comment:
Since `pub fn get_clock(clock_id: &ClockID) -> Result<&Arc<dyn Clock>>` is defined in `syscall/clock_gettime.rs`, I think the definition of `ClockId` should also be there.